src/styles/blocks/header.scss

```diff
@@ -0,0 +1,60 @@
+$img-width: 40px;
+$header-height: 60px;
+$header-bar-height: 4px;
+$header-main-text-color: #060b35;
+$header-secondary-text-color: #00acdc;
```

Comment:
Move variables to the variables file, some colors are related not only for header and used for other elements